doc/reference/devices_disk.md

```diff
@@ -1,58 +1,61 @@
-either be a bind-mount of an existing file or directory on the host, or
-if the source is a block device, a regular mount.
+Disk devices are essentially mount points inside the instance.
+They can either be a bind-mount of an existing file or directory on the host, or, if the source is a block device, a regular mount.
 
 They can also be created by {ref}`attaching a storage volume to an instance <storage-attach-volume>`.
 
 LXD supports the following additional source types:
 
-- Ceph RBD: Mount from existing Ceph RBD device that is externally managed. LXD can use Ceph to manage an internal file system for the instance, but in the event that a user has a previously existing Ceph RBD that they would like use for this instance, they can use this command.
+Ceph RBD
+: Mount an existing Ceph RBD device that is externally managed.
 
-  Example command:
+  LXD can use Ceph to manage an internal file system for the instance, but if you have an existing Ceph RBD that you would like to use for an instance, you can add it with the following command:
 
-  ```
-  lxc config device add <instance> ceph-rbd1 disk source=ceph:<my_pool>/<my-volume> ceph.user_name=<username> ceph.cluster_name=<username> path=/ceph
-  ```
+      lxc config device add <instance_name> <device_name> disk source=ceph:<pool_name>/<volume_name> ceph.user_name=<user_name> ceph.cluster_name=<cluster_name> path=<path_in_instance>
 
-- CephFS: Mount from existing CephFS device that is externally managed. LXD can use Ceph to manage an internal file system for the instance, but in the event that a user has a previously existing Ceph file system that they would like use for this instance, they can use this command.
+CephFS
+: Mount an existing CephFS device that is externally managed.
 
-  Example command:
+  LXD can use Ceph to manage an internal file system for the instance, but if you have an existing Ceph file system that you would like to use for an instance, you can add it with the following command:
 
-  ```
-  lxc config device add <instance> ceph-fs1 disk source=cephfs:<my-fs>/<some-path> ceph.user_name=<username> ceph.cluster_name=<username> path=/cephfs
-  ```
+      lxc config device add <instance_name> <device_name> disk source=cephfs:<fs_name>/<path> ceph.user_name=<user_name> ceph.cluster_name=<cluster_name> path=<path_in_instance>
 
-- VM cloud-init: Generate a cloud-init configuration ISO from the `user.vendor-data`, `user.user-data` and `user.meta-data` configuration keys and attach to the VM so that cloud-init running inside the VM guest will detect the drive on boot and apply the configuration. Only applicable to virtual-machine instances.
+VM `cloud-init`
+: Generate a `cloud-init` configuration ISO from the `user.vendor-data`, `user.user-data` and `user.meta-data` configuration keys and attach it to the VM, so that `cloud-init` running inside the VM detects the drive on boot and applies the configuration.
```

Comment:
This should probably use the more recent `cloud-init.` prefix rather than `user.`
See https://linuxcontainers.org/lxd/docs/master/reference/instance_options/#instance-options